config_defaults_inc.php

```diff
@@ -2085,6 +2085,28 @@
 $g_bug_closed_status_threshold = CLOSED;
 
 /**
+ * Bug becomes closed for submission of automated bug reports (attaches)
+ * if its status is >= $g_bug_collect_reports_threshold. The bug starts 
+ * accept more bug reports again if its status becomes less than this 
+ * threshold. The default is ASSIGNED. 
+ * Read-only, closed and fixed bugs are closed for bug report submission 
+ * regardless of this option.
+ * See also $g_bug_max_auto_reports option.
+ * See also SOAP.mc_issue_report function.
+ * @global integer $g_bug_collect_reports_status_threshold
+ */
+$g_bug_collect_reports_status_threshold = ACKNOWLEDGED;
```

Comment:
Here is why it is ACKNOWLEDGED: one can set max report to a very large number, may be even almost unlimited. Then they start to use their app and Mantis. All will be more or less fine, but only up until they made a really bad mistake, which will appear on almost all of their clients. 

Now what? Devs will receive insane amount of reports and could do nothing to stop it - until administrator will change max reports to lower value. If you have this value to something other than resolved, then devs can simply acknowledge the bug and stop reports from being accepted.

Surely, this would be a mistake in Mantis configuration made by administrator. He should not increase limit without changing this option too. But why not offer additional safe net?

Besides, from purely logic view: if you acknowledge the bug, why would you want to receive more reports?

P.S. Just in case, if it is unclear: only files (attaches) are rejected. Count (votes) will still be added - up until issue is resolved or become read-only.